meson.build

```diff
@@ -0,0 +1,182 @@
+project(
+  'cloud-init',
+  meson_version: '>=0.63.0', # rockylinux/9
+  license: 'GPL-3 or Apache-2.0',
+  default_options: [
+    # The default can yield broken results.
+    'python.install_env=auto',
+  ],
+)
+system = host_machine.system()
+sysconfdir = get_option('sysconfdir')
+INIT_SYSTEM = get_option('init_system')
```

Comment:
This variable name is capitalized. Please match the naming style of the other variables.